config.json

```diff
@@ -296,6 +296,7 @@
       {
         "cv.build": "source",
         "cv.sentry": false,
+        "qx.dynlocale": false,
```

Comment:
Deactivating the dynamic locale switching is fine, as we do not need it in the client. But out of curiosity: Did you experience any problems with this feature here, that made you change the setting?

Reply:
Yes, there were some problems (cf. Gitter conversation).
The code is expecting a String in some places but only got the translation-object instead.